### ISSUE TITLE
Frontend sign up flow

### ISSUE THREAD

**Issue by vinielk** [CodetheChangeFoundation/dda#15]
**Acceptance Criteria**
* Create form for users to sign up (i.e. create a new account)
* All users who sign up should be default users (not employers or admin)
* User must verify their email before they are able to login (this setup should be done in AWS Cognito, just need to make sure it works as expected)

**Notes**
* Blocked by #4 (Setup AWS Cognito)

**Out of scope**
* `Forgot password?` flow is out of scope for this ticket and will be addressed in a future ticket
